twitter_ads/enum.py

```diff
@@ -77,7 +77,8 @@ def enum(**enums):
     LINE_ITEM='LINE_ITEM',
     PROMOTED_TWEET='PROMOTED_TWEET',
     ORGANIC_TWEET='ORGANIC_TWEET',
-    TARGETING_CRITERION='TARGETING_CRITERION'
+    TARGETING_CRITERION='TARGETING_CRITERION',
```

Comment:
What is this one for? `ENTITY` should be for analytics entity types.

Reply:
leaving this for now as it just came from the existing code